src/interfaces/IExecuteReadOnlyResponse.ts

```diff
@@ -1,13 +1,6 @@
-import { IEvent } from "./IEvent";
-import { ISlot } from "./ISlot";
+import { IExecuteReadOnlyData } from "./IExecuteReadOnlyData";
 
 export interface IExecuteReadOnlyResponse {
-    /// The slot at which the read-only execution occurred.
-	executed_at: ISlot;
-    /// The result fo the read-only execution.
-	result: string | null;
-    /// The output events generated by the read-only execution.
-	output_events: Map<string, IEvent>;
-    /// The gas cost for the execution
-    gas_cost: number;
+	returnValue: Uint8Array;
```

Comment:
Why does this not match the API and have a wrapper structure like that ?

Reply:
Answered on DIscord : The api response is in the info: IExecuteReadOnlyData. The idea is to have the returnValue ready for the user to use rather than to extract it from the IExecuteReadOnlyData. If one needs some further data contained inside IExecuteReadOnlyData, the info can be always inspected.